butterknife-gradle-plugin/src/main/java/butterknife/plugin/ButterKnifePlugin.kt

```diff
@@ -7,6 +7,7 @@ import com.android.build.gradle.FeaturePlugin
 import com.android.build.gradle.LibraryExtension
 import com.android.build.gradle.LibraryPlugin
 import com.android.build.gradle.api.BaseVariant
+import groovy.util.XmlSlurper
```

Comment:
I hate absolutely everything about this type's FQCN: Groovy, "util", XML, and "Slurper"...